benchcab/benchcab.py

```diff
@@ -131,6 +133,9 @@ def fluxsite_submit_job(self) -> None:
                 storage_flags=[],  # TODO(Sean) add storage flags option to config
                 verbose=self.args.verbose,
                 skip_bitwise_cmp="fluxsite-bitwise-cmp" in self.args.skip,
+                benchcab_path=self.benchcab_exe_path
+                if self.benchcab_exe_path
+                else "benchcab",
```

Comment:
Is this really the behaviour we want when calling `render_job_script`? If `self.benchcab_exe_path` evaluates to False, shouldn't we instead raise an error? Isn't it a sign of something that has gone wrong within benchcab?

Reply:
Yes we should probably raise an error here since it means something has gone wrong with the path. This was a bit of a hack to get the unit tests to pass since `self.benchcab_exe_path` is always `None` unless we have installed benchcab. The right way is probably to specify `self.benchcab_exe_path` as an argument so we can mock it in the tests.